docs/user-guide/installation.md

```diff
@@ -117,7 +123,7 @@ autoload -U compinit && compinit
 
 To make this change permanent, the above commands can be added to `~/.profile` file.
 
-[Release]: https://github.com/terraform-docs/terraform-docs/releases
-[Homebrew]: https://brew.sh
-[Scoop]: https://scoop.sh/
-[Chocolatey]: https://www.chocolatey.org
+[release]: https://github.com/terraform-docs/terraform-docs/releases
+[homebrew]: https://brew.sh
+[scoop]: https://scoop.sh/
+[chocolatey]: https://www.chocolatey.org
```

Comment:
Hmm, these should be case sensitive. Was there a reason you changed them?

Reply:
Weird, I didn't even touch those. Pretty sure, my VS Code uses a plugin to format code. May be that caused it. Lemme change it to original formatting.